### DISCUSSION TITLE
Store Data to S3

### DISCUSSION THREAD

**Discussion by travisdillard** [ckan/ckan#6630] [Q&A]
Anyone have any luck getting CKAN to store datasets to an S3 bucket?  I am using the EC2 marketplace CKAN AMI provided by LinkDigital and I've tried many of the extensions such as CloudConnector, S3-resources, and 2 others.  None of them install properly and cause CKAN to return a 500 error code.  Looking for tips on the best way to integrate CKAN on AWS using as many AWS native resources as possible.  Thanks!

**Comment by jqnatividad** [comment]
Have you tried https://github.com/TkTech/ckanext-cloudstorage?  I use it all the time without issues.
